drange/lib/DataStructure.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
```

Comment:
I used the originally mentioned npm package and I update the name. Is there anything else?